.github/workflows/ci.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9"]
-        nautobot-version: ["1.0.1"]
+        nautobot-version: ["1.3.3"]
```

Comment:
Due to the way testing changed between 1.2.11 and 1.3.0, I don't have the desire to support the dual testing infrastructure. Do you think it would be worthwhile?

Reply:
Added 1.2.11 in Matrix, removed 3.10 for now in the matrix.